src/components/tutorial.ts

```diff
@@ -1,98 +1,99 @@
 import { GameScore } from "../data/game-score";
 
 export class Tutorial {
-    public width: number;
-    public height: number;
-    public context: CanvasRenderingContext2D;
-    public tutorialImg: any;
-    public imagesLoaded: boolean = false;
-    public targetStonePositions: any;
-    public startx: number;
-    public starty: number;
-    public endx: number;
-    public endy: number;
-    public endTutorial: boolean = false;
-    public puzzleNumber: number;
-    x: number;
-    y: number;
-    dx: number;
-    dy: number;
-    absdx: number;
-    absdy: number;
+  public width: number;
+  public height: number;
+  public context: CanvasRenderingContext2D;
+  public tutorialImg: any;
+  public imagesLoaded: boolean = false;
+  public targetStonePositions: any;
+  public startx: number;
+  public starty: number;
+  public endx: number;
+  public endy: number;
+  public endTutorial: boolean = false;
+  public puzzleNumber: number;
+  x: number;
+  y: number;
+  dx: number;
+  dy: number;
+  absdx: number;
+  absdy: number;
 
-    constructor(context, width, height, puzzleNumber) {
```

Comment:
Can we separate the formatting changes next time so it will be easier to review actual changes?
Merging it for now though.